frontend/src/app/forms/steps/field-designation-page/field-designation-page.component.html

```diff
@@ -0,0 +1,41 @@
+<div class="main-description">
+  <h1>Industries and Positions</h1>
+  <span>
+    So that we can know in which industries and positions you are intrested,
+    please choose from the following.
+  </span>
+</div>
+
+<ion-item class="ion-no-padding">
+  <ion-label color="medium">Industry</ion-label>
+  <ion-select
+    multiple
+    [formControl]="formsData.controls.fieldDesignationInfo.controls.field"
+    [interfaceOptions]="{ cssClass: 'custom-popover' }"
+    interface="popover"
+  >
+    <ion-select-option
+      *ngFor="let field of industryArray"
+      [value]="field.value"
+    >
+      {{ field.displayName }}</ion-select-option
+    >
+  </ion-select>
+</ion-item>
+
+<ion-item class="ion-no-padding">
+  <ion-label color="medium">Position</ion-label>
+  <ion-select
+    multiple
+    [formControl]="formsData.controls.fieldDesignationInfo.controls.designation"
+    [interfaceOptions]="{ cssClass: 'custom-popover' }"
+    interface="interface"
```

Comment:
`interface="interface"` looks wrong to me :)

Reply:
Yep this is an error. But on this error i could now see, that the interface="popover" was unnecessary here too, because both popover dropmenus looked the same. :) I delete this line too.